ravem/indico_ravem/plugin.py

```diff
@@ -48,26 +43,27 @@ class SettingsForm(IndicoForm):  # pragma: no cover
     polling_interval = IntegerField(_('Polling interval'), [NumberRange(min=1000)],
                                     description=_('The delay between two polls in ms, at least 1000 ms<br>'
                                                   '(delete the cached var.js to take effect)'))
+    room_feature = StringField(_('Room feature'), [DataRequired()],
```

Comment:
`QuerySelectField`, see the `room_feature` settings in other plugins

you also need to add this in the plugin class in order for the setting to work:

```
    settings_converters = {
        'room_feature': ModelConverter(RoomFeature),
    }
```